src/libstd/sys/unix/thread.rs

```diff
@@ -20,6 +20,11 @@ use time::Duration;
 
 use sys_common::thread::*;
 
+#[cfg(not(target_os = "l4re"))]
+pub const default_min_stack_size: usize = 2 * 1024 * 1024;
+#[cfg(target_os = "l4re")]
+pub const default_min_stack_size: usize = 1024 * 1024;
```

Comment:
Conventionally in rust constants have full uppercase names like `DEFAULT_MIN_STACK_SIZE`